### PR TITLE
[LICENSE] Avoid using no license kyuubi ascii logo

### PR DESCRIPTION
**Pull request by alex601049502**
alex601049502 [Closes #529](https://github.com/yaooqinn/kyuubi/pull/529)    Target Issue Test Plan [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

close #529

### CODE REVIEW

File: bin/kyuubi [hunk 1]
```diff
@@ -209,3 +209,15 @@ case $1 in
...
+echo "  __  __                           __"
+echo " /\ \/\ \                         /\ \      __ "
+echo " \ \ \/'/'  __  __  __  __  __  __\ \ \____/\_\ "
+echo "  \ \ , <  /\ \/\ \/\ \/\ \/\ \/\ \\\\\ \ '__\`\/\ \ "
+echo "   \ \ \\\\\\\`\\\\\ \ \_\ \ \ \_\ \ \ \_\ \\\\\ \ \L\ \ \ \ "
+echo "    \ \_\ \_\/\`____ \ \____/\ \____/ \ \_,__/\ \_\ "
+echo "     \/_/\/_/\`/___/> \/___/  \/___/   \/___/  \/_/ "
+echo "                /\___/ "
+echo "                \/__/ "
+}
```
Comment (pan3793) [inline bin/kyuubi:223]: Nit: new line at end of file

File: bin/kyuubi [hunk 2]
```diff
@@ -209,3 +209,15 @@ case $1 in
     usage
     ;;
 esac
+
+function echo_logo() {
```
Comment (pan3793) [inline bin/kyuubi:213]: change func name to `kyuubi_logo`

**Comment by pan3793** [comment]
Please also delete `bin/kyuubi-logo` file

File: bin/kyuubi [hunk 3]
```diff
@@ -209,3 +209,15 @@ case $1 in
...
+echo "  __  __                           __"
+echo " /\ \/\ \                         /\ \      __ "
+echo " \ \ \/'/'  __  __  __  __  __  __\ \ \____/\_\ "
+echo "  \ \ , <  /\ \/\ \/\ \/\ \/\ \/\ \\\\\ \ '__\`\/\ \ "
+echo "   \ \ \\\\\\\`\\\\\ \ \_\ \ \ \_\ \ \ \_\ \\\\\ \ \L\ \ \ \ "
```
Comment (pan3793) [inline bin/kyuubi:218]: This line seems like not correct.

Comment (alex601049502) [reply bin/kyuubi:218]: Because the  double '\' need to transfer meaning.

**Comment by turboFei** [comment]
how about add license into origin logo file.
And then `cat kyuubi-logo |grep -v '^#'`.

**Comment by pan3793** [comment]
Close this in favor #532

**Pull request closed without merging**